core/server/web/api/canary/admin/routes.js

```diff
@@ -42,6 +42,7 @@ module.exports = function apiRoutes() {
     router.get('/integrations', mw.authAdminApi, http(apiCanary.integrations.browse));
     router.get('/integrations/:id', mw.authAdminApi, http(apiCanary.integrations.read));
     router.post('/integrations', mw.authAdminApi, http(apiCanary.integrations.add));
+    router.post('/integrations/:id/api_key/:keyid/refresh', mw.authAdminApi, http(apiCanary.integrations.edit));
```

Comment:
@kevinansfield Had done a quick check with Hannah about the endpoint and this works structurally ^

Reply:
No concerns 👍